cmd/gen/main.go

```diff
@@ -129,8 +129,8 @@ func main() {
 const lineStartsWith = "func (c *Client) "
 
 var nameRE = regexp.MustCompile("^\\w+\\(")
-var argsRE = regexp.MustCompile("^.+\\)\\s\\(")
-var returnRE = regexp.MustCompile("^.+\\)")
+var argsRE = regexp.MustCompile("^.+\\)\\s[\\(e]")
+var returnRE = regexp.MustCompile("^.+\\s{")
```

Comment:
💡 Mostly an idea/suggestion for possible future improvement, definitely not a current action item: we can use AST parsing to perform code generation in a possibly more maintainable way in the future: https://eli.thegreenplace.net/2021/rewriting-go-source-code-with-ast-tooling/

Reply:
Yes, this "generator" is a disgrace. :disappointed:

Reply:
Gets the work done is what matters 🥇